internal/serialization/serialization.go

```diff
@@ -182,14 +218,19 @@ func (s *Service) registerDefaultSerializers() error {
 	if err = s.registerSerializer(portableSerializer); err != nil {
 		return err
 	}
-	s.nameToID["!portable"] = ConstantTypePortable
+	s.nameToID["!portable"] = TypePortable
```

Comment:
I think we can get rid of the `nameToID` altogether by storing the few serializers we put it in the service object

Reply:
Removed it at: https://github.com/hazelcast/hazelcast-go-client/pull/581/commits/646f1893fc76711d3ffecc2c6d09d0253044d316